pyscf/mcpdft/tfnal_derivs.py

```diff
@@ -86,8 +86,9 @@ def eval_ot(otfnal, rho, Pi, dderiv=1, weights=None, _unpack_vot=True):
             first functional derivative of Eot wrt (density, pair
             density) and their derivatives. If _unpack_vot = True, shape
             and format is ([a, ngrids], [b, ngrids]) : (vrho, vPi);
-            otherwise, [c, ngrids] : [rho,Pi,|rho'|^2,rho'.Pi',|Pi'|^2]
+            otherwise, [c, ngrids] : [rho,Pi,|rho'|^2,tau,rho'.Pi',|Pi'|^2]
             ftGGA: a=4, b=4, c=5
+            tmGGA: a=5, b=1, c=4 (drop Pi')
             tGGA: a=4, b=1, c=3 (drop Pi')
             *tLDA: a=1, b=1, c=2 (drop rho')
```

Comment:
Can we clarify "drop tau" in ftGGA, tGGA, and *tLDA?

Reply:
Fixed.